src/ui/components/shared/MaterialIcon.tsx

```diff
@@ -18,6 +18,23 @@ type MaterialIconProps = React.HTMLProps<HTMLDivElement> & {
   iconSize?: keyof typeof SIZE_STYLES;
 };
 
+let isChecking = false;
+function useMaterialIconCheck() {
+  useEffect(() => {
+    if (isChecking) {
+      return;
+    }
+    isChecking = true;
+    let id = setInterval(() => {
+      if (typeof document === "object" && document.fonts.check("12px Material Icons")) {
```

Comment:
I would probably do `(document as any).fonts` here to appease the typescript gods. You could also declare fonts on window, but it doesn't seem particularly useful.